net/remote/remote-node.sh

```diff
@@ -77,6 +80,20 @@ local|tar)
       export SOLANA_CUDA=1
     fi
     set -x
+    rm -rf ./solana-node-keys
+    rm -rf ./solana-node-stakes
+    mkdir ./solana-node-stakes
+    if [[ -n $stakeNodesInGenesisBlock ]]; then
+      for i in $(seq 1 "$numNodes"); do
+        solana-keygen new -o ./solana-node-keys/"$i"
```

Comment:
Add an -f here I think, `solana-keygen new -f -o ./solana-node-keys/"$i"`, to handle `./net.sh restart`

Reply:
seems -f is not a valid option..

Reply:
https://github.com/solana-labs/solana/blob/master/keygen/src/keygen.rs#L35-L39

Reply:
Somehow it gave me an error when I tried to run in on testnet.

Reply:
```solana-keygen new -f -o ./solana-node-keys/0
error: Found argument '-f' which wasn't expected, or isn't valid in this context
```